tests/automation_framework/README.md

```diff
@@ -6,13 +6,33 @@ This is to validate the response of worker, work order and work
 order receipts APIs as per Enterprise Ethereum Alliance Off-Chain 
 Trusted Compute Specification v1.1.
 
-Steps to execute automation test framework
+Steps to execute automation test framework:
 
-1. Build Avalon: Docker-based build (recommended)
-	docker-compose up --build
-2. Open a Docker container shell using following command 
-	docker exec -it tcf bash
-3. Run automation
-	cd $TCF_HOME/tests/validation_suite
-	pytest (All tests)
-	pytest work_order_tests/test_work_order_success.py (single test)
+1. Build Avalon: Refer to Build.md from the link below.
+     https://github.com/hyperledger/avalon/blob/master/BUILD.md
+2. Open a Docker container shell using following command. 
+     docker exec -it avalon-shell bash
+4. Running automation tests in Listener/SDK/Proxy Mode(Ethereum/Fabric) using markers.
```

Comment:
Add a new line before the Listener mode starts.

Reply:
done